packages/appstore/src/modules/tour-guide/wallet-tour-guide.tsx

```diff
@@ -1,40 +1,44 @@
 import React from 'react';
-import Joyride, { StoreState } from 'react-joyride';
-import { Localize } from '@deriv/translations';
+import Joyride, { EVENTS, StoreState } from 'react-joyride';
 import { useStore, observer } from '@deriv/stores';
 import { getWalletStepConfig, getWalletStepLocale, wallet_tour_styles } from 'Constants/tour-steps-config';
 
 const WalletTourGuide = observer(() => {
     const { traders_hub } = useStore();
     const { is_wallet_tour_open, toggleIsWalletTourOpen } = traders_hub;
+    const [scroll_offset, setScrollOffset] = React.useState<number>(200);
```

Comment:
@aizad-deriv In my opinion for basic types like `boolean`, `number` and `string` we don't need to set type for `useState` because TS can get based on the default value. But you can ignore it :)